### ISSUE TITLE
[Addon] StardockFences like movable squares for grouping

### ISSUE THREAD

**Issue by 42willow** [Ju1-js/25th-hour-dynamic#34]
Some moveable and resizable boxes for grouping applications.

Multiple themes in a drop down menu?

**Comment by Ju1-js** [comment]
I'm not sure how I'd get the boxes' shapes and sizes to persist, I'll look into it later.
*I think I remember hearing something about cookies ...maybe.*

**Comment by 42willow** [comment]
https://github.com/rocksdanister/lively/issues/1199

**Comment by Ju1-js** [comment]
Is there a specific style you want it to look like?

**Comment by 42willow** [comment]
Not really, I don't have a preference. I can make a few options but it might be a few days before I have enough time to work on this. Maybe a minimalistic one which matches the theme to start? Just a transparent box with some rounded corners would be nice.

**Comment by 42willow** [comment]
https://en.wikipedia.org/wiki/9-slice_scaling
This could be used to scale the squares with styling without distortion - found it on the aseprite wiki

This issue is also related to #35 - as it's pretty much the same with a text box in the middle instead of the top for the title
